### ISSUE TITLE
Add Accessibility Testing to PRs

### ISSUE THREAD

**Issue by jostut** [WTW-IM/es-components#527]
https://github.com/marketplace/axe-linter

Can someone add the above app? 

This will check for accessibility issues before merging in new work. The rules it checks against are also used in Lighthouse. Should be noted Accessibility testing should still be tested manually, but this will help catch any common issues

**Comment by stevematney** [comment]
We can't use this app because they don't have a plan available for Organization accounts and repositories. It can only be used on personal repos.

We can probably do something to improve a11y testing by utilizing [`axe-core` for react running in jest](https://github.com/dequelabs/axe-core/tree/develop/doc/examples/jest_react). We already run jest for our unit tests, so adding something like this won't need any extra lift to ensure that Pull Requests are tested for accessibility in CI.

**Comment by jostut** [comment]
Oh nvm, I thought this project was Open-source.

**Comment by stevematney** [comment]
This project is open source, but the Axe Linter doesn't seem to have an option to run on anything but personal repos. This is open source, but it's owned by the Github Organization. 😄
